File: C++/Programming Basics with C++/10. For-loop/simple_tasks/NumberSequence.cpp
#include <iostream>

using namespace std;

int main() {
    int n;
    cin >> n;

    int num;
    cin >> num;

    int min;
    int max;
    min = num;
    max = num;

    for (int i = 0; i < n - 1; i++) {
        cin >> num;

        if (num < min) {
            min = num;
        }

        if (num > max) {
            max = num;
        }
    }

    cout << "Max number: " << max << endl;
    cout << "Min number: " << min << endl;

    return 0;
}

File: C#/Programming Basics with C#/08. While-Loop/04. Number sequence/NumberSequence.cs
using System;

class NumberSequence
{
    static void Main()
    {
        var a = Console.ReadLine();

        if (a == "END")
        {
            return;
        }

        int maxNum = int.Parse(a);
        int minNum = int.Parse(a);

        while (a != "END")
        {
            int num = int.Parse(a);
            if (maxNum < num)
            {
                maxNum = num;
            }
            if (minNum > num)
            {
                minNum = num;
            }
            a = Console.ReadLine();
        }
        Console.WriteLine($"Max number: {maxNum}");
        Console.WriteLine($"Min number: {minNum}");
    }
}
